app/views/measures/_measures.html.erb

```diff
@@ -25,21 +25,24 @@
   <div class="govuk-tabs__panel" id="import">
     <h2 class="govuk-heading-l"><%= measures_heading(anchor: 'import') %></h2>
 
-    <%= render partial: 'declarables/consigned', locals: { declarable: declarable } %>
+    <%= render 'declarables/consigned', declarable: declarable %>
 
     <% if declarable.import_measures.any? %>
-      <%= render partial: 'measures/grouped/navigation', locals: { uk_declarable: uk_declarable, xi_declarable: xi_declarable } %>
+      <%= render 'measures/grouped/navigation',
+                 uk_declarable: uk_declarable,
+                 xi_declarable: xi_declarable
+      %>
 
       <% if TradeTariffFrontend::ServiceChooser.xi? %>
-        <%= render partial: 'measures/grouped/xi',
-                   locals: { uk_declarable: uk_declarable,
-                             xi_declarable: xi_declarable,
-                             rules_of_origin_schemes: rules_of_origin_schemes || roo_all_schemes } %>
+        <%= render 'measures/grouped/xi',
+                   uk_declarable: uk_declarable,
+                   xi_declarable: xi_declarable,
+                   rules_of_origin_schemes: rules_of_origin_schemes || roo_all_schemes %>
```

Comment:
As discussed on slack, I am just cleaning up the view, removing unnecessary partials/locals as discussed.
Sometimes they are necessary, for example, when using collections.

Rule: when unnecessary, remove them.

Reply:
:raised_hands: